Octokit.Tests.Integration/Clients/RepositoryInvitationsClientTests.cs

```diff
@@ -70,6 +179,158 @@ public class TheGetAllForCurrentMethod
                 Assert.NotNull(invitations.FirstOrDefault(i => i.Repository.Id == response.Repository.Id));
             }
         }
+
+        [IntegrationTest]
+        public async Task ReturnsCorrectCountOfInvitationsWithStart()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoNames = Enumerable.Range(0, 6).Select(i => Helper.MakeNameWithTimestamp($"public-repo{i}")).ToList();
```

Comment:
Let's use pagesize=1 and just create 2 test repos

Reply:
Done.